docs/clients/dovecot.txt

```diff
@@ -36,7 +36,7 @@ Configuring Dovecot to work with Virtual Exim
 
       driver = mysql
       connect = host=/var/run/mysqld/mysqld.sock dbname=vexim user=vexim password=CHANGE
-      default_pass_scheme = CRYPT
+      default_pass_scheme = SHA512-CRYPT #BLF-CRYPT (<- this is for bcrypt, only in *BSD!) #PLAIN
```

Comment:
Yes, you are right this should work. This makes life much easier.
glibc >= 2.7 is needed, so no real restriction.